systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -186,7 +187,7 @@ private static void waitForIoTProjectDeleted(Kubernetes kubernetes, IoTProject p
     }
 
     public static boolean isIoTInstalled(Kubernetes kubernetes) {
-        return kubernetes.getCRD("iotprojects.iot.enmasse.io") != null;
+        return kubernetes.getCRD(IoTCrd.project().getMetadata().getName()) != null;
```

Comment:
I believe this check might not be accurate (depending on what its used for), because the iot project CRD is installed even if IoT is not installed: https://github.com/EnMasseProject/enmasse/tree/master/templates/crds

Reply:
Good point. But then I think @famartinrh or @kornys should take a look. I was simply trying to re-use an existing constant, rather than adding a new string.

Reply:
good point I will look at it

Reply:
it will be fixed in different PR